cli/template/extras/src/utils/api.ts

```diff
@@ -3,15 +3,15 @@ import { createTRPCNext } from "@trpc/next";
 import { type inferRouterInputs, type inferRouterOutputs } from "@trpc/server";
 import superjson from "superjson";
 
```

Comment:
some kind of comment here to explain what this is, similar to what we've done in server/trpc.ts?

Reply:
done, see https://github.com/t3-oss/create-t3-app/pull/934/commits/f633df2220065fe74fdbaa98d3c8be7419c156c7